### ISSUE TITLE
Fertilizer Stage

### ISSUE THREAD

**Issue by Jayshua83** [Timmeey86/FS22_ConservationAgriculture#17]
Is the Fertilizer Stage Option able to have an option for allowing it too apply to the second stage if you want instead of first?

**Comment by Timmeey86** [comment]
There is an option which directly gives you the second stage fertilization (when not using Precision Farming).
Is that what you want, or were you thinking about adding +1, so if it wasn't fertilized, set it to first stage, and if it was at first stage already, set it to second?

**Comment by Jayshua83** [comment]
Without PF I have 3 options for FERT.

First Stage, FULL Fert, None!

**Comment by Jayshua83** [comment]
Was hoping if it was easy to have either a +1 or a Second Stage option in case we sprayed/spread first!

**Comment by Timmeey86** [comment]
Could you explain how the "Second Stage" option would work exactly?

**Comment by Jayshua83** [comment]
Honestly this may not be worth your time! We will just change our Prep Order as and do it properly! lol :)

**Comment by Timmeey86** [comment]
Ah well the topic of conservation agriculture is quite controversial, especially in Europe, which is why I'm aiming for high configurability so everyone can adjust it to what they think is right ;-)
I'll try and see if I can get +1 working now. I tried in the past but ended up manipulating the same spot more than once, so would always set it to max. But I made a different change later which should no longer process the same spot twice in general, so it's worth a look if it works now

**Comment by Jayshua83** [comment]
Our original Pre-CA order was : Lime > Fertilize > Mulch > Fertilize > Seed/Plant > Pre Growth Herbicide

Our original CA order was : Lime > Fertilize > Seed Oilseed > Plant Primary (which mulched but didn't fertilize since we already did it...

Our Current Order is : Lime > Seed Oilseed > Plant Primary > Spray Fert for 2nd round. (which honestly makes sense... as my wife said you probably wouldn't fertilize Pre-cover since those nutrients would be waste on the cover crop.

**Comment by Timmeey86** [comment]
Makes sense. In real life, you would have to use additional fertilizer when transforming to CA and you might even have to fertilize your cover crops in the beginning (to get more biomass out of it). According to some, If you manage to get a balanced ecosystem in your field (above and below ground combined), you can get to a point where you don't have to fertilize at all, but that's kind of impossible to simulate with just two fertilizer stages

**Comment by Timmeey86** [comment]
![grafik](https://github.com/Timmeey86/FS22_ConservationAgriculture/assets/10223149/10369a43-378c-414d-85bc-03f078b32faa)
Yeah unfortunately it looks like I can'T make it work properly. This should only be the first level, but some parts are being processed twice by the looks of it

**Comment by Timmeey86** [comment]
Implemented in branch https://github.com/Timmeey86/FS22_ConservationAgriculture/tree/feature/fixed-fertilization
